moler/util/moler_test.py

```diff
@@ -30,34 +31,55 @@ def steps_end():
         MolerTest._was_steps_end = True
 
     @staticmethod
-    def error(msg, raise_exception=False):
+    def error(msg, dump=None, raise_exception=False):
```

Comment:
Please do not change order of parameters in method header because it may cause fail of other parts of Moler. Only if really necessary :)

Reply:
Done